server/cron/calendar_cron_test.go

```diff
@@ -242,7 +241,7 @@ func TestCalendarEventsMultipleHosts(t *testing.T) {
 					Integrations: fleet.TeamIntegrations{
 						GoogleCalendar: &fleet.TeamGoogleCalendarIntegration{
 							Enable:     true,
-							WebhookURL: webhookServer.URL,
+							WebhookURL: "https://foo.example.com",
```

Comment:
Nit. I typically use a constant if I'm going to reuse the same string multiple places.